src/schema/ghga.yaml

```diff
@@ -1165,13 +1148,19 @@ classes:
         required: true
         inlined: true
         inlined_as_list: false
-      has data use condition:
+      has data use permission:
+        description: >-
+          Data use permission associated with a policy. Typically one or more terms from DUO
+          and should be descendants of 'DUO:0000001 data use permission'.
+        required: true
```

Comment:
Data Use Permission is marked as optional in the spreadsheet (0.9.1), but here it is a required field...

Reply:
We changed the spreadsheet field to mandatory.